### DISCUSSION TITLE
Тулкит

### DISCUSSION THREAD

**Discussion by pfzim** [Arkady23/V380-Web#16] [Q&A]
Добрый день!

Расскажите, как получилось скомпилировать веб сервер под эту камеру. Можете поделиться тулкитом для сборки?
Я пытался собрать [motion](https://github.com/Motion-Project/motion) под неё, но у меня ничего не вышло.

**Comment by Arkady23** [comment]
Я использовал Htttp-сервер от alienatedsec/yi-hack-v4. Он подходит к огромному количеству разных камер. Для тех у кого старый Linux пришлось изменять имена библиотек. После такого изменения httpd запустился, однако rtsp — нет. Я думаю, что и httpd от [alienatedsec/yi-hack-v5](https://github.com/alienatedsec/yi-hack-v5) будет совместим. Компилятор был выложен еще в самом первом проекте [shadow-1/yi-hack-v3](https://github.com/shadow-1/yi-hack-v3/).

**Reply by pfzim** [reply]
Не вижу компилятора в проекте. Тыкните меня носом.
А можете попробовать собрать motion в своем окружении?

**Reply by Arkady23** [reply]
https://github.com/shadow-1/yi-hack-v3/issues/23
